pageserver/benches/bench_layer_map.rs

```diff
@@ -178,12 +183,17 @@ fn build_layer_map() -> LayerMap {
 }
 
 fn large_layer_map(c: &mut Criterion) {
-    let layer_map = build_layer_map();
+    // A list of layer filenames, extracted from our performance test environment, from
+    // a project where we have run pgbench many timmes. The pgbench database was initialized
+    // between each test run.
+    let layer_map = build_layer_map(PathBuf::from("benches/odd-brook-layernames.txt"));
 
     c.bench_function("search", |b| {
```

Comment:
Why do we need this test for hardcoded key lookup if below  we perform search for real keys?